Documentation/ProjectFile/prj/process_variables/process_variable/boundary_conditions/boundary_condition/Dirichlet/t_strict_bc_domain.md

```diff
@@ -0,0 +1,11 @@
+This option, if set to false, allows the geometrical boundary mesh for a given
+variable (and component) not strictly to coincide with the definition domain of
+the variable.
+
+The tag is optional as is set to true by default (strict mode).
+
+In case of a process variable being defined only on a part of a mesh, as, for
+example in LIE, where the displacement jump variable is defined only on the
+fractures, or in case of hydro mechanics, where the pressure variable is defined
+on lower order elements, the boundary mesh is not strictly required to coincide
+with the part of the process variable's boundary part.
```

Comment:
Actually it's not checked if they coincide, but only if they have the same number of nodes, is it?

Reply:
It is assumed that geometrically the boundary mesh is fitting the bulk mesh. This is not checked, but "guaranteed" if the constructMeshesFromGeometries tool is used.
Now, if the strict_bc_domain is set, then all nodes of the bc_mesh for that variable must have a valid entry in the DOF table.

Reply:
That's what I wanted to say. :smile:

And I'd add the following to the documentation:
> It is assumed that geometrically the boundary mesh is fitting the bulk mesh.